site/docs/v1/tech/apis/_webhook-request-body.adoc

```diff
@@ -48,6 +48,15 @@ The HTTP basic authentication username that is sent as part of the HTTP request
 [field]#webhook.readTimeout# [type]#[Integer]# [required]#Required#::
 The read timeout in milliseconds used when FusionAuth sends events to the Webhook.
 
+[field]#webhook.signatureConfiguration# [type]#[Object]# [optional]#Optional# [since]#Available since 1.48.0#::
```

Comment:
`webhook.eventsEnabled` is another example of specifying an optional Object.  I think this is technically more correct, but you're right that we don't always list the base object.

Reply:
We never (as far as I know) document the object - just the properties. 

So in this case, we would document two field:

```
webhook.signatureConfiguration.enabled
webhook.signatureConfiguration.signingKeyId
```

You should be able to compare to any other API with nested fields. Application and Tenant have lots of them.

Reply:
Looks like we only document objects for open ended fields like `webhook.eventsEnabled` and `webhook.data`.

Added since tags and removed the `webhook.signatureConfiguration` object field.